Latest commit on Aug 1st seems to break build

Overnight our build process broke
```
[root]# make docker
mkdir -p bin
docker run --rm \
-e TARGET_GOOS= \
-e TARGET_GOARCH= \
-v '/home/centos/amazon-ecr-credential-helper/bin':/go/src/github.com/awslabs/amazon-ecr-credential-helper/bin \
sha256:4e88df0997089a874e3aab1eb346291f553423d6880de7f515f3631c411a3448
./scripts/build_binary.sh ./bin/local 0.7.0 44fa0cd
# golang.org/x/sys/unix
vendor/golang.org/x/sys/unix/syscall.go:83:16: undefined: unsafe.Slice
vendor/golang.org/x/sys/unix/syscall_linux.go:1018:20: undefined: unsafe.Slice
vendor/golang.org/x/sys/unix/syscall_linux.go:2297:9: undefined: unsafe.Slice
vendor/golang.org/x/sys/unix/syscall_unix.go:118:7: undefined: unsafe.Slice
vendor/golang.org/x/sys/unix/sysvshm_unix.go:33:7: undefined: unsafe.Slice
make: *** [Makefile:44: bin/local/docker-credential-ecr-login] Error 2
make: *** [docker] Error 2
```

Previous commit 8c2948d9a0fe82c1151ce8f7a31d1143bae76fb6 does seem to work.
Running on `CentOS Linux release 7.9.2009 (Core)`

--- a/.github/workflows/build.yaml
+++ b/.github/workflows/build.yaml
@@ -27,7 +27,13 @@ jobs:
         with:
           go-version: ${{ matrix.go }}
       - run: make get-deps
-        if: ${{ matrix.go >= '1.17' }}
+        if: ${{ matrix.go >= '1.19' }}
+
+      # test Dockerfile build on Linux. macOS runners do not ship with Docker
+      # installed by default whereas Ubuntu runners do.
+      - name: Build in Dockerfile
+        run: make docker
+        if: ${{ matrix.os == 'ubuntu-22.04' }}
 
       # Apple Silicon is not supported by Go < 1.16.
       # https://go.dev/blog/go1.16

--- a/Dockerfile
+++ b/Dockerfile
@@ -11,7 +11,7 @@
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
 
-FROM golang:1.15
+FROM golang:1.19
 
 WORKDIR /go/src/github.com/awslabs/amazon-ecr-credential-helper
 

